### PR TITLE
iotop: add page

### PR DESCRIPTION
**Pull request by einverne**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/linux/iotop.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+- Only show processes or threads actually doing I/O:
+
+`iotop -o`
+
+- Show I/O usage in non-interactive mode:
+
+`iotop -b`
+
+- Only I/O usage of processes(Normally iotop shows all threads):
```
Comment (schneiderl) [inline pages/linux/iotop.md]: Please rephrase this to be a verb. I'm unsure of what this actually does so I'm unable to give you a suggestion on how to do it haha.

File: pages/linux/iotop.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+`iotop -P`
+
+- Show I/O usage of given PID(s):
+
+`iotop -p {{PID}}`
+
+- Show I/O usage of given USER:
```
Comment (schneiderl) [inline pages/linux/iotop.md]: I think we can have this (and the example) in lower case.

**Comment by agnivade** [comment]
Doesn't macos have this too ? If so, this should be in common.

**Comment by einverne** [comment]
I have not used in macos before, but I do a search and it seems work in macos. So I made the change. But I have not verified this commands in macos personally.

File: pages/common/iotop.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+- Show only processes or threads actually doing I/O:
+
+`iotop -o`
+
+- Show I/O usage in non-interactive mode:
+
+`iotop -b`
+
+- Show only I/O usage of processes(Normally iotop shows all threads):
```
Comment (agnivade) [inline pages/common/iotop.md]: (Normally iotop shows all threads) => (Default is to show all threads)

File: pages/common/iotop.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+`iotop -P`
+
+- Show I/O usage of given PID(s):
+
+`iotop -p {{PID}}`
+
+- Show I/O usage of given user:
```
Comment (agnivade) [inline pages/common/iotop.md]: of _a_ given user
